### PR TITLE
Additions

### PR DESCRIPTION
**Pull request by GuptaEesh**
- added search functionality
- added 404 page

### CODE REVIEW

File: src/Screens/pomodoro-app-screen.js [hunk 1]
```diff
@@ -9,17 +9,20 @@ import { IoCheckmarkDoneCircleSharp } from "react-icons/io5";
...
 import { Button } from "../components";
 import { useData } from "../helper";
+import { EmptyPage } from "./empty-page";
 import "./screen.css";
 const PomoDoroApp = ({ setTask, setIsModalOpen }) => {
   const { todoState, dispatchToDo } = useData();
-  const { todo } = todoState;
+  const { todo, searchTerm } = todoState;
   const [selectedTag, setSelectedTag] = useState("All");
+  console.log(searchTerm);
```
Comment (Hamdrive) [inline src/Screens/pomodoro-app-screen.js]: cleanup `console.log()`

**Review by Hamdrive — COMMENTED** [review]
Nice work implementing Search and 404 page and empty page
Good check for empty search result
Overall like the code written, clean and simple

File: src/Screens/pomodoro-app-screen.js (hunk 1 above)
Comment (GuptaEesh) [reply src/Screens/pomodoro-app-screen.js]: Thanks for pointing that out. Cleaning it

File: src/components/organisms/header/header.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            inputClass="input-text md"
+            inputType="text"
+            inputPlaceHolder="Search"
+            inputValue={value}
+            inputFunc={inputHandler}
+          />
+          <BsSearch
+            onClick={openSearch}
+            className="position-absolute position-right search-icon cursor-pointer"
+          />
```
Comment (Varun-28) [inline src/components/organisms/header/header.jsx:38]: - Understandable code with precise implementation
- code is well-formatted
- clean css
- ready to merge

**Pull request merged**